client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -1351,6 +1297,42 @@ public void testGetDataFrameAnalyticsConfig_ConfigNotFound() {
         assertThat(exception.status().getStatus(), equalTo(404));
     }
 
+    public void testStartDataFrameAnalyticsConfig() throws Exception {
+        String sourceIndex = "test-source-index";
+        createIndexWithDefaultMapping(sourceIndex);
+        IndexRequest doc = new IndexRequest(sourceIndex);
+        doc.source("{\"total\": 10000}", XContentType.JSON);  // We need at least one doc in the index to perform the analysis
+        highLevelClient().index(doc, RequestOptions.DEFAULT);
+        String destIndex = "test-dest-index";  // This index must not exist, otherwise the reindexing step will fail
+
+        assertTrue(highLevelClient().indices().exists(new GetIndexRequest(sourceIndex), RequestOptions.DEFAULT));
+        assertFalse(highLevelClient().indices().exists(new GetIndexRequest(destIndex), RequestOptions.DEFAULT));
+
+        MachineLearningClient machineLearningClient = highLevelClient().machineLearning();
+        String configId = "test-config";
+        DataFrameAnalyticsConfig config = DataFrameAnalyticsConfig.builder(configId)
+            .setSource(new DataFrameAnalyticsSource(sourceIndex))
+            .setDest(new DataFrameAnalyticsDest(destIndex))
+            .setAnalysis(new OutlierDetection())
+            .build();
+
+        execute(
+            new PutDataFrameAnalyticsRequest(config),
+            machineLearningClient::putDataFrameAnalytics, machineLearningClient::putDataFrameAnalyticsAsync);
+
+        AcknowledgedResponse startDataFrameAnalyticsResponse = execute(
+            new StartDataFrameAnalyticsRequest(configId),
+            machineLearningClient::startDataFrameAnalytics, machineLearningClient::startDataFrameAnalyticsAsync);
+        assertTrue(startDataFrameAnalyticsResponse.isAcknowledged());
+
+        // TODO(przemyslaw.witek): Instead of sleeping, poll for task state "STOPPED" once GetDataFrameAnalyticsStats API method is
+        //                         implemented in MachineLearningClient
+        Thread.sleep(10 * 1000);
```

Comment:
Apparently it breaks connections in unrelated tests. 
I'll solve it this way: I'll wait for dest index creation with timeout. When GetStats is implemented, I'll change the implementation to wait for STOPPED.

Reply:
As suggested by @davidkyle, I'll remove the test for now as currently there is no easy way to make it correct.